p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1962,7 +1962,7 @@ Unlike C, P4 allows a very limited number of implicit casts. The reason is that
 Implicit casts are allowed in P4 only when their meaning is completely unambiguous:
 
 - To convert an ```int``` value to a fixed-width type.
-- In assignments when the right-hand side has a different type from left-hand side.
+- In assignments when the right-hand side has type ```int```.
```

Comment:
@jnfoster - The current p4c gives a warning about the RHS not being able to fit in the LHS, which makes me happy.  If you want, I can add a sentence in the spec recommending that the compiler should give a warning in this case.